go.sum

```diff
@@ -47,6 +47,7 @@ github.com/kr/text v0.1.0 h1:45sCR5RtlFHMR4UwH9sdQ5TC8v0qDQCHnXt+kaKSTVE=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
 github.com/lithammer/dedent v1.1.0 h1:VNzHMVCBNG1j0fh3OrsFRkVUwStdDArbgBWoPAffktY=
 github.com/lithammer/dedent v1.1.0/go.mod h1:jrXYCQtgg0nJiN+StA2KgR7w6CiQNv9Fd/Z9BP0jIOc=
+github.com/mattn/go-runewidth v0.0.9 h1:Lm995f3rfxdpd6TSmuVCHVb/QhupuXlYr8sCI/QdE+0=
```

Comment:
I don't think the changes in this `go.sum` belong in this PR.

Reply:
Removed these changes. One note, without them when I try to run the tests I get the following error:
```
$ go test ./...
go: github.com/jhump/protoreflect@v1.8.1 requires
        google.golang.org/grpc@v1.27.0 requires
        golang.org/x/oauth2@v0.0.0-20180821212333-d2e6202438be: missing go.sum entry; to add it:
        go mod download golang.org/x/oauth2
```
Should this be addressed separately?

Reply:
Yes, they'll be handled in a separate PR.